CMakeLists.txt

```diff
@@ -253,6 +253,7 @@ define_sdl_subsystem(Hidapi)
 define_sdl_subsystem(Power)
 define_sdl_subsystem(Sensor)
 define_sdl_subsystem(Dialog)
+define_sdl_subsystem(Tray)
```

Comment:
If this does depend on the video driver, this needs to be:
```
define_sdl_subsystem(Tray DEPS SDL_VIDEO)
```

Reply:
It does not depend on the video driver.

Reply:
I saw too late testtray used the window for lifetime management.
Can that be done without windows?